inc/options/class-wpseo-options.php

```diff
@@ -81,6 +79,30 @@ public static function get_instance() {
 		return self::$instance;
 	}
 
+	/**
+	 * Registers an option to the options list.
+	 *
+	 * @param string       $option_name     The option name to register.
+	 * @param WPSEO_Option $option_instance Instance of the option.
+	 */
+	public static function register_option( $option_name, WPSEO_Option $option_instance ) {
+		if ( $option_instance->multisite_only && ! self::is_multisite() ) {
+			unset( self::$options[ $option_name ], self::$option_names[ $option_name ] );
+
+			return;
+		}
+
+		if ( ! array_key_exists( $option_name, self::$options ) ) {
+			self::$options[ $option_name ] = get_class( $option_instance );
```

Comment:
Code Climate indicates that this line wouldn't be covered in unit tests yet?

Reply:
I think the lines are added by accident. I will check later where I can add tests for the entire class where possible.